src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -160,12 +159,7 @@ export const ExampleObjectLinks = (props: BrowserProps) => {
 
     return (
       <div key={exampleObject.object_id} className={styles.exampleLink}>
-        <Link to={path}>
-          <span className={styles.objectType}>
-            {upperFirst(exampleObject.type)}
-          </span>
-          <span className={styles.objectLabel}>{exampleObject.label}</span>
-        </Link>
+        <Link to={path}>{`Example ` + exampleObject.type}</Link>
```

Comment:
if you are using backticks here, then why not `Example ${exampleObject.type}`?

on the other hand, why not `<Link to={path}>Example {exampleObject.type}</Link>`? That's even more idiomatic for jsx.

Reply:
That's done